docs/Buttons/GetCapabilities/assets/GetCapabilities.txt

```diff
@@ -0,0 +1,62 @@
+participant App
+participant SDL
+participant HMI
+
+HMI->SDL:BC.OnReady
+SDL->HMI:Buttons.IsReady
+HMI-->SDL:Buttons.IsReady (code:0)
+SDL->SDL:  Check availability\nof `HMICapabilitiesCacheFile`
+
+opt Using cached data from `HMICapabilitiesCacheFile` - Happy Path 
+
+note over SDL:<align:center>`HMICapabilitiesCacheFile`\n**is not** available
```

Comment:
@Mariia-Novosiadla `opt Using cached data from HMICapabilitiesCacheFile - Happy Path ` for me this implies that the data from the cache will be used, but then I see that `HMICapabilitiesCacheFile\n**is not** available` - it is confusing